src/main/fc/fc_core.c

```diff
@@ -848,6 +848,8 @@ bool processRx(timeUs_t currentTimeUs)
     pidSetAcroTrainerState(IS_RC_MODE_ACTIVE(BOXACROTRAINER) && sensors(SENSOR_ACC));
 #endif // USE_ACRO_TRAINER
 
+    pidSetAntiGravityState(IS_RC_MODE_ACTIVE(BOXANTIGRAVITY) || feature(FEATURE_ANTI_GRAVITY));
```

Comment:
Very minor: More efficient for default settings when `feature(FEATURE_ANTI_GRAVITY)` is checked first.

Reply:
I'll change this around.  I simply moved this check from fc_rc.c so it was previously this ordering.